platform-cloud/docs/studios/add-studio.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Add a Studio"
+description: "Add a Studio in Platform."
+date created: "2025-09-04"
+last updated: "2025-10-21"
+tags: [data, session, studios]
+---
+
+Select the **Studios** tab, and then select **Add Studio**.
+
+:::info[**Prerequisites**]
+You will need the following to get started:
+
+- Valid credentials for accessing cloud storage resources
+- **Maintain** role permissions (minimum)
+- A compute environment with sufficient resources (scale based on data volume):
+    - **CPUs**: 2 (minimum)
+    - **Memory**: 8192 MB  (minimum)
+- [Data Explorer](../data/data-explorer) enabled
+:::
+
+:::tip[**Access and permissions**]
+If you're unable to see the Studios tab, contact your Platform administrator.
+:::
```

Comment:
Is this still relevant? With the switch to Usage-based Pricing, isn't Studios now turned on for everyone?